yarn-project/noir-protocol-circuits/src/crates/types/src/abis/complete_address.nr

```diff
@@ -1,11 +1,11 @@
 use crate::point::Point;
-use crate::address::Address;
+use crate::address::AztecAddress;
 use crate::hash::{compute_partial_address,compute_contract_address_from_partial};
 
 struct CompleteAddress{
-    address : Address,
+    address : AztecAddress,
     public_key : Point, 
-    // TODO(David): Can we type this as Address instead of Field?
+    // TODO(David): Can we type this as AztecAddress instead of Field?
```

Comment:
Should there be a type for the partial making it more clear what it actually is, because it is pretty up hill to figure out atm.

Reply:
Seems like a good idea. Created an [issue](https://github.com/AztecProtocol/aztec-packages/issues/3682).